src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -532,7 +532,7 @@ public async Task BackupDatabaseOnce()
 
                     var t = Database.Operations.AddOperation(
                         null,
-                        $"Database backup: {Database.Name}",
+                        $"Manual backup for database: {Database.Name}",
```

Comment:
Maybe `Manual database backup` ?

Reply:
Currently, it looks like this:
![image](https://user-images.githubusercontent.com/17376974/94034393-a6177e00-fdca-11ea-8d8e-cda50f610932.png)

imho - it makes more sense to have:
_Manual backup for database: q1_

do you still prefer:
_Manual database backup: q1_
?

Reply:
ok let's leave it